feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/domain/locks/reusable/LocksChange.kt

```diff
@@ -22,3 +24,15 @@ fun MutableList<ReusableLock>.addIfPositive(type: ReusableLock.Type, amount: Bal
         add(ReusableLock(type, amount))
     }
 }
+
+fun List<LocksChange>.maximize(): LocksChange {
+    val maxLockedAmountChange = this.maxByOrNull { it.lockedAmountChange.absoluteDifference() }
+    val maxLockedPeriodChange = this.maxByOrNull { it.lockedPeriodChange.absoluteDifference() }
+    val maxTransferableChange = this.maxByOrNull { it.transferableChange.absoluteDifference() }
```

Comment:
The maximum transferrable change comes from the max locked amount change, given that changes are positive

Reply:
Agree but for more readable version I suggest to keep the logic in this state